package.json

```diff
@@ -17,7 +17,7 @@
     "prerelease": "gulp clean-release",
     "release": "export NODE_ENV=prod && gulp test:e2e && npm run build && gulp release && node gulp/release-tasks/node-template-flatten.js && gulp replace-links",
     "postrelease": "gulp clean-redundant-release",
-    "start": "npm run build && gulp watch",
+    "start": "yarn build && gulp watch",
```

Comment:
npm or yarn, most of the build process is npm based so we should keep it that way to reduce issues.